jsonpatcher/jsonpatcher.go

```diff
@@ -9,31 +9,20 @@ import (
 	"time"
 
 	jsonpatch "github.com/evanphx/json-patch"
-	ds "github.com/ipfs/go-datastore"
 	cbornode "github.com/ipfs/go-ipld-cbor"
 	format "github.com/ipfs/go-ipld-format"
 	logging "github.com/ipfs/go-log"
 	"github.com/multiformats/go-multihash"
 	core "github.com/textileio/go-threads/core/store"
 )
 
-type operationType int
-
-const (
-	create operationType = iota
-	save
-	delete
-)
-
 var (
-	log                           = logging.Logger("jsonpatcher")
-	errSavingNonExistentInstance  = errors.New("can't save nonexistent instance")
-	errCantCreateExistingInstance = errors.New("cant't create already existent instance")
-	errUnknownOperation           = errors.New("unknown operation type")
+	log                 = logging.Logger("jsonpatcher")
+	errUnknownOperation = errors.New("unknown operation type")
 )
 
 type operation struct {
-	Type      operationType
+	Type      core.ActionType
```

Comment:
Switched to using core.ActionType here, as there are multiple nearly identical versions of ActionType floating around, likely due to past scenarios where the event store was its own package.

Reply:
Makes sense to me. Let's keep the two types here @carsonfarmer.

Reply:
Ok, that explanation makes sense. Will revert.